src/app/gps/qgsgpstoolbar.cpp

```diff
@@ -322,3 +373,49 @@ void QgsGpsToolBar::destinationMenuAboutToShow()
   }
 }
 
+void QgsGpsToolBar::createLocationWidget()
+{
+  mLocationButton = new QToolButton();
+  mLocationButton->setToolTip( tr( "Current GPS Location" ) );
```

Comment:
You'll hate me  for suggesting this but shouldn't the button be named mInformationButton, and its tooltip be "Current GPS Location Information" ? That'd better reflect what this is about, and better match the appropriately named Qgis::GpsInformationComponent class name.

Reply:
Done!